### PR TITLE
publish HumanSkeletonArray in human_pose_estimator

### PR DESCRIPTION
**Pull request by mqcmd196**
This PR publishes HumanSkeletonArray and it may enable to visualize HumanSkeleton in rviz with https://github.com/jsk-ros-pkg/jsk_recognition/pull/2769.

### CODE REVIEW

**Comment by mqcmd196** [comment]
@iory Could you review?

File: python/coral_usb/human_pose_estimator.py [hunk 1]
```diff
@@ -59,6 +62,8 @@ def __init__(self, model_file=None, namespace='~'):
         self.label_names = ['human']
 
         # publishers
+        self.pub_skel = self.advertise(
+            namespace + 'output/skels', HumanSkeletonArray, queue_size=1)
```
Comment (knorth55) [inline python/coral_usb/human_pose_estimator.py:66]: skletons is better

**Comment by mqcmd196** [comment]
Thank you, updated

File: python/coral_usb/human_pose_estimator.py [hunk 2]
```diff
@@ -163,25 +169,36 @@ def image_cb(self, msg):
...
                         Pose(position=Point(x=key_x, y=key_y)))
+            for i in range(len(pose_msg.poses)):
+                bone_name = "{}->{}".format(pose_msg.limb_names[i - 1],
+                                            pose_msg.limb_names[i])
+                skel_msg.bone_names.append(bone_name)
+                seg = Segment()
+                seg.start_point = pose_msg.poses[i - 1].position
+                seg.end_point = pose_msg.poses[i].position
+                skel_msg.bones.append(seg)
+                skels_msg.human_ids.append(Int32(data=i))
```
Comment (knorth55) [inline python/coral_usb/human_pose_estimator.py:194]:
```suggestion
            skels_msg.human_ids.append(Int32(data=i))
```

Comment (mqcmd196) [reply python/coral_usb/human_pose_estimator.py]: Sorry...

**Review by iory — APPROVED** [review]

**Pull request merged**